tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb

```diff
@@ -0,0 +1,1778 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "TypeError",
+     "evalue": "__init__() got an unexpected keyword argument 'sampling_method'",
+     "output_type": "error",
+     "traceback": [
+      "\u001B[0;31m---------------------------------------------------------------------------\u001B[0m",
+      "\u001B[0;31mTypeError\u001B[0m                                 Traceback (most recent call last)",
+      "\u001B[0;32m<ipython-input-59-a294184dc293>\u001B[0m in \u001B[0;36m<module>\u001B[0;34m\u001B[0m\n\u001B[0;32m----> 1\u001B[0;31m numeric_metric_range_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder = NumericMetricRangeMultiBatchParameterBuilder(\n\u001B[0m\u001B[1;32m      2\u001B[0m     \u001B[0mname\u001B[0m\u001B[0;34m=\u001B[0m\u001B[0;34m\"column_mean_range\"\u001B[0m\u001B[0;34m,\u001B[0m\u001B[0;34m\u001B[0m\u001B[0;34m\u001B[0m\u001B[0m\n\u001B[1;32m      3\u001B[0m     \u001B[0mmetric_name\u001B[0m\u001B[0;34m=\u001B[0m\u001B[0;34m\"column.mean\"\u001B[0m\u001B[0;34m,\u001B[0m\u001B[0;34m\u001B[0m\u001B[0;34m\u001B[0m\u001B[0m\n\u001B[1;32m      4\u001B[0m     \u001B[0msampling_method\u001B[0m\u001B[0;34m=\u001B[0m\u001B[0;34m\"bootstrap\"\u001B[0m\u001B[0;34m,\u001B[0m\u001B[0;34m\u001B[0m\u001B[0;34m\u001B[0m\u001B[0m\n\u001B[1;32m      5\u001B[0m     \u001B[0mmetric_domain_kwargs\u001B[0m\u001B[0;34m=\u001B[0m\u001B[0mdomain\u001B[0m\u001B[0;34m.\u001B[0m\u001B[0mdomain_kwargs\u001B[0m\u001B[0;34m,\u001B[0m\u001B[0;34m\u001B[0m\u001B[0;34m\u001B[0m\u001B[0m\n",
+      "\u001B[0;31mTypeError\u001B[0m: __init__() got an unexpected keyword argument 'sampling_method'"
+     ]
+    }
+   ],
+   "source": [
+    "numeric_metric_range_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder = NumericMetricRangeMultiBatchParameterBuilder(\n",
+    "    name=\"column_mean_range\",\n",
+    "    metric_name=\"column.mean\",\n",
+    "    sampling_method=\"bootstrap\",\n",
+    "    metric_domain_kwargs=domain.domain_kwargs,\n",
+    "    false_positive_rate=1.0e-2,\n",
+    "    round_decimals=0,\n",
+    "    data_context=data_context,\n",
+    "    batch_request=multi_batch_batch_request,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameter_container: ParameterContainer = ParameterContainer(parameter_nodes=None)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "numeric_metric_range_parameter_builder.build_parameters(\n",
+    "    parameter_container=parameter_container,\n",
+    "    domain=domain,\n",
+    "    parameters = {domain.id: parameter_container}\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(parameter_container.parameter_nodes)"
```

Comment:
Assuming you are already thinking about this, but just a reminder that it would be good to commit the executed version of this cell as you've done with the others once the error is resolved.